### PR TITLE
Fix for random JUnit failures in framework tests

### PR DESCRIPTION
**Pull request by cheenamalhotra**
Fixes random test failures due to unordered Select Queries in framework tests.

Although there are similar implementations in other tests which need to be fixed, this PR focuses on the tests that have been observed failing and are designed using framework package.

### CODE REVIEW

File: src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyISQLServerBulkRecordTest.java [hunk 1]
```diff
@@ -96,7 +96,11 @@ public void testISQLServerBulkRecord() throws SQLException {
                     if (JDBCType.BIT == sqlType.getJdbctype()) {
                         CurrentRow[j] = ((0 == ThreadLocalRandom.current().nextInt(2)) ? Boolean.FALSE : Boolean.TRUE);
                     } else {
-                        CurrentRow[j] = sqlType.createdata();
+                        if(j==0) {
+                            CurrentRow[j] = i+1;
```
[the remark this answers is not in the document]
Comment (cheenamalhotra) [reply src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyISQLServerBulkRecordTest.java]: Tried that, but it requires IDENTITY_INSERT to be enabled in order to make TVP and Bulk Copy work, with lots of test changes. I don't think it's worth the effort if we have to eventually enable IDENTITY_INSERT.

File: src/test/java/com/microsoft/sqlserver/testframework/DBColumn.java [hunk 2]
```diff
@@ -35,6 +35,13 @@
     public String getColumnName() {
         return columnName;
     }
+    
+    /**
+     * @return Escaped "columnName"
+     */
+    public String getEscapedColumnName() {
```
[the remark this answers is not in the document]
Comment (cheenamalhotra) [reply src/test/java/com/microsoft/sqlserver/testframework/DBColumn.java]: Our table columns in tests don't contain either, so it has no impact in our tests.

File: src/test/java/com/microsoft/sqlserver/testframework/DBColumn.java [hunk 3]
```diff
@@ -90,4 +97,14 @@ Object getRowValue(int row) {
         return columnValues.get(row);
     }
 
-}
+    /**
+     * Updates rowIndexes to all rows
+     * @param totalRows number of rows
+     */
+    public void populateRowId(int totalRows) {
```
[the remark this answers is not in the document]
Comment (cheenamalhotra) [reply src/test/java/com/microsoft/sqlserver/testframework/DBColumn.java]: This function and `populateValues` above in the same file, populate data in 'columnValues' which are inserted iteratively in the table creation process. This is first time data insertion, so can't be fetched from anywhere.

File: src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyColumnMappingTest.java [hunk 4]
```diff
@@ -186,7 +186,7 @@ public void testRepetativeCM() {
         }
 
         // add column mapping for duplicate column in destination
-        bulkWrapper.setColumnMapping(1, 24);
```
Comment (peterbae) [inline src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyColumnMappingTest.java]: Why did we increment this by one?

Comment (peterbae) [reply src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyColumnMappingTest.java]: nvm, got it

**Pull request merged**